### ISSUE TITLE
This Static Web App already has the maximum number of staging environments (10)

### ISSUE THREAD

**Issue by bowdenk7** [Azure/static-web-apps#941]
**Describe the bug**

Hitting the limit (10 in my case) fails CI. This is pretty frustrating. I really want a way to increase this limit, which I would happily pay for, but in the meantime just don't fail CI. These are preview deployments. It's a luxury, not a requirement. So if you can't provision more than 10, so be it, but don't fail my CI.

**Expected behavior**
Don't fail CI - leave a warning or comment on the PR or something.

**Comment by ABL6COB** [comment]
Yes, we are facing this issue and need to manually delete the preview environments when it hits 10 count.
As suggested, a warning would be fine in case new environment cannot be created and the CI should not fail.

**Comment by bowdenk7** [comment]
Closing as duplicate of #765

**Comment by bowdenk7** [comment]
Actually I'm going to re-open this. A better fix here is to increase the limit, but the big problem that makes this un-usable is it fails CI.

**Comment by ABL6COB** [comment]
I agree to @bowdenk7, we need a way to make sure the pipeline is not failing since it has reached the maximum preview environments.

**Comment by ABL6COB** [comment]
Any updates here? We are facing this issue and blocking us from using preview environments.

**Comment by MTCMarkFranco** [comment]
Any update on this?

**Comment by thomasgauvin** [comment]
Updating this thread as well, we've heard the feedback and have been putting in the infrastructural improvements to have increased limits across the board for Static Web Apps, whether that be number of staging envs, file counts, app sizes, etc. We're still some months away from this at this time, I will continue to provide status updates in this thread.

We're tracking this in https://github.com/Azure/static-web-apps/issues/765

**Comment by bowdenk7** [comment]
This issue is not concerned with increasing the limit (though I agree that is a better solution). In the meantime, simply not failing CI would make this service usable. Until then, I will use other solutions.

**Comment by thomasgauvin** [comment]
@bowdenk7 can you provide additional details as to why you need the CI to be successful despite not deploying? The failing CI serves as an indication that the environment did not get deployed. Do you store artifacts instead of deploying, or do you rely on the same workflow to give an indication of tests passing?
